platforms/access-bridge/src/endpoints.ts

```diff
@@ -0,0 +1,20 @@
+import { IncomingMessage, ServerResponse } from 'node:http';
+
+import { AccessController } from './controllers/access-controller.js';
+
+export type EndpointHandler = {
```

Comment:
This is an elegant, simple implementation, but did you look at using a framework like express or nestjs? Since this is a demo repo, using a framework would help keep the code to just what's interesting for demo purposes. While this is a nice implementation, it adds complexity for someone trying to go through and understand what the JWP specific integration code should look like.

Reply:
I did introduce express, but still not seeing any differences in addressing your concern. If you have any other suggestions how we should define routes, feel free to recommend. I think it's very nice to have them separately defined like this.